index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Haliljon</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container"
+        ><a href="#" class="link">Welcome</a>
```

Comment:
You did a good job👌but you can improve it by following the exact Figma design in which the welcome text is bold, please make sure to update this.